gssutils/scrapers/govuk.py

```
@@ -284,4 +287,48 @@ def eth_facts_service(scraper, tree):
             pass
 
 
+def content_api_guidance(scraper, tree):
+    
+    content_api_url = f'https://www.gov.uk/api/content/{scraper.uri.replace("https://www.gov.uk/", "")}'
+    r = scraper.session.get(content_api_url)
+    if r.status_code != 200:
+        raise Exception(f'Failed to get url "{content_api_url}", status_code "{r.status_code}".')
+    page_json = r.json()
+
+    if 'title' in page_json:
```

this pattern ends up as a silent fail which we'd rather avoid really (as-in this would work if the title is missing - which is good, but we'd want it screaming at us telling us whats happening under the hood), so something like:
```
title = metadata.get("title", None)
if title is None:
    logging.warning(f'The title for  dataset {scraper.url} not set, title field missing from content api')
 else:
   dataset.title = title
```
etc for the other dataset level fields.